### PR TITLE
[ADD] estate: basic directory structure T#69469

### PR DESCRIPTION
**Pull request by Suavesito-Olimpiada**
@deivislaya

Can you check this PR? It's ready for review.

Regards!

### CODE REVIEW

**Comment by deivislaya** [comment]
Please. Execute pre-commit-vauxoo script

File: estate/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+{
+    'name': 'Real Estate',
+    'version': '1.0',
```
Comment (deivislaya) [inline estate/__manifest__.py]:
```suggestion
    'version': '16.0.0.0.0',
```

File: estate/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    'description': """Real Estate module""",
+    'author': 'Vauxoo',
+    'license': 'GPL-3 or any later version',
```
Comment (deivislaya) [inline estate/__manifest__.py:7]:
```suggestion
    'license': 'OPL-1',
```

File: estate/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    'depends': ['base'],
+    'application': True,
+    'installable': True,
+    'maintainer': 'Suavesito-Olimpiada',
```
Comment (deivislaya) [inline estate/__manifest__.py:11]: Please delete key. In this exercise, the __manifest__.py file should only define the name and the dependencies of our modules. The only necessary framework module for now is base.

**Review by deivislaya — CHANGES_REQUESTED** [review]

File: estate/__manifest__.py (hunk 1 above)
Comment (Suavesito-Olimpiada) [reply estate/__manifest__.py]: According to the documentation of OCA on [version numbers](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers) "`12.0.1.0.0` is expected for the first release of an `12.0` module", so this module should be `16.0.1.0.0` and move forward following the "semantics `breaking.feature.fix`". In [this commit](https://github.com/OCA/crm/blame/f462ddef4993f79917dd54b51177f9177389d74f/crm_location/__manifest__.py) you can see that the first ever released version of the module `crm_location` was version `10.0.1.0.0`.

File: estate/__manifest__.py (hunk 2 above)
Comment (Suavesito-Olimpiada) [reply estate/__manifest__.py:7]: I'll change it.

File: estate/__manifest__.py (hunk 3 above)
Comment (Suavesito-Olimpiada) [reply estate/__manifest__.py:11]: I'll clean it to the minimum.

**Pull request closed without merging**